sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/InferWindowGroupLimit.scala

```diff
@@ -58,8 +61,9 @@ object InferWindowGroupLimit extends Rule[LogicalPlan] with PredicateHelper {
    */
   private def isExpandingWindow(
       windowExpression: NamedExpression): Boolean = windowExpression match {
-    case Alias(WindowExpression(_, WindowSpecDefinition(_, _,
-    SpecifiedWindowFrame(RowFrame, UnboundedPreceding, CurrentRow))), _) => true
```

Comment:
We can't delete these directly, please do not affect the origin logic.

Reply:
You can add a parameter to distinguish.

Reply:
I have also found problems with this implementation before, I will make changes later.